src/main/java/pl/arturzgodka/controllers/ItemsController.java

```diff
@@ -0,0 +1,79 @@
+package pl.arturzgodka.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import pl.arturzgodka.datamodel.ItemArmorDataModel;
+import pl.arturzgodka.datamodel.ItemDataModel;
+import pl.arturzgodka.datamodel.ItemWeaponDataModel;
+import pl.arturzgodka.jsonmappers.ItemMapper;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+@Controller
+public class ItemsController {
+
+    private final ItemMapper itemMapper = new ItemMapper();
+
+    @RequestMapping("/itemsTypes.html")
+    public String getItemTypes(Model model) {
+        model.addAttribute("itemTypesList", ItemClassesAndNamesLists.itemTypes);
+        return "itemsTypes";
+    }
+
+    @RequestMapping(value="/{itemTypes}", method = RequestMethod.GET)
+    public String getItemTypesNames(Model model, @PathVariable(value="itemTypes") String itemType) {
+
+        Set<String> itemTypeNames = ItemClassesAndNamesLists.getItemTypesNames(itemType);
+        model.addAttribute("selectedItemType", itemType);
+        model.addAttribute("itemTypesNames", itemTypeNames);
+
+        return "itemTypes";
+    }
+
+    /*@RequestMapping("/Armor/")
```

Comment:
why the commented out code?

Reply:
That was first version of controller logic for displaying items list but it was divided into 2 methods. Now it is merged into 1 method to operate on both request: for displaying armor and weapon items lists.
Done, this part of commented code is out.